platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/SerializablePublicKey.java

```diff
@@ -28,6 +30,7 @@
 import java.security.spec.InvalidKeySpecException;
 import java.security.spec.X509EncodedKeySpec;
 import java.util.Arrays;
+import org.apache.commons.lang3.builder.ToStringBuilder;
```

Comment:
100% in favor of dropping dependencies like this if we can. We currently use `StringBuilder` all over the place... perhaps it would be better to tackle this in a follow-up PR?

Reply:
I didn't consider `ToStringHelper` mostly because it will require dependency on Guava library (if I understand correctly what you're talking about). We use `ToStringBuilder` in multiple places in the code anyway, so I considered it to be a standard. 

Also, even though you're right about the performance of `ToStringBuilder`, I'm not sure if it's a significant concern for this method that is normally helpful for debug purposes.

Reply:
I'd prefer to see us using String.formatted, as that's probably the clearest to read (and requires no dependencies), but I can see value in making that change in stages.